src/app/loadout-builder/filter/PerkPicker.tsx

```diff
@@ -262,7 +226,28 @@ function PerkPicker({
   return (
     <Sheet
       onClose={onClose}
-      header={header}
+      header={
```

Comment:
Moved it and deleted the header buttons like I did in the mod picker. There aren't many perks for exotics so the buttons aren't necessary imo.